docusaurus.config.ts

```diff
@@ -44,6 +44,9 @@ const config: Config = {
         theme: {
           customCss: './src/css/custom.css',
         },
+        gtag: {
+          trackingID: "G-W07RNQP7NT",
```

Comment:
Should we also add `anonymizeIP: true` to avoid privacy/GDPR issues?

Reply:
Good point. Thank you Christian